app/javascript/components/patient/assessment/actions/LastDateExposure.js

```diff
@@ -226,51 +222,40 @@ class LastDateExposure extends React.Component {
                   ariaLabel="Last Date of Exposure Input"
                   isClearable
                 />
+                {/* <Row className="pt-2">
+                  <Col> */}
+                    <OverlayTrigger
+                      key="tooltip-ot-ce"
+                      placement="left"
+                      overlay={
+                        <Tooltip id="tooltip-ce" style={this.props.patient.monitoring ? { display: 'none' } : {}}>
+                          Continuous Exposure cannot be turned on or off for records on the Closed line list. If this monitoree requires monitoring due to a
+                          Continuous Exposure, you may update this field after changing Monitoring Status to &quot;Actively Monitoring&quot;
+                        </Tooltip>
+                      }>
+                      <span className="d-inline-block">
+                        <Form.Check
+                          size="lg"
+                          label="CONTINUOUS EXPOSURE"
+                          id="continuous_exposure"
+                          disabled={!this.props.patient.monitoring}
+                          checked={this.state.continuous_exposure}
+                          onChange={() => this.openContinuousExposureModal()}
+                        />
+                      </span>
+                    </OverlayTrigger>
+                    <InfoTooltip tooltipTextKey="continuousExposure" location="right"></InfoTooltip>
+                  {/* </Col>
+                </Row> */}
```

Comment:
Was this supposed to be uncommented or removed?

Reply:
fixed: https://github.com/SaraAlert/SaraAlert/pull/895/commits/7f93c59be1e1dfac1bc02adb3a0ed63d68c24210